pyproject.toml

```diff
@@ -56,6 +56,7 @@ filterwarnings = [
     "ignore::DeprecationWarning",
     'ignore:Tasks created via `Application\.create_task` while the application is not running',
     "ignore::ResourceWarning",
+    "ignore:Hey. You seem to be using the `python-telegram-bot-raw` library:UserWarning",
```

Comment:
when does this trigger in the tests? It shouldn't trigger right?

Reply:
Good point, will have to investigate...